Document/0x04f-Testing-Network-Communication.md

```diff
@@ -42,15 +42,19 @@ In these cases you need to monitor and analyze the network traffic first in orde
 
 - Route the traffic through the host machine. You can set up your machine as the network gateway, e.g. by using the built-in Internet Sharing facilities of your operating system. You can then use [Wireshark](https://www.wireshark.org "Wireshark") to sniff any traffic from the mobile device;
 
-- Sometimes you need to execute a MITM attack to force the mobile device to talk to you. For this scenario you should consider [bettercap](https://github.com/bettercap/bettercap "bettercap") to redirect network traffic from the mobile device to your host machine (see below);
+- Sometimes you need to execute a MITM attack to force the mobile device to talk to you. For this scenario you should consider [bettercap](https://github.com/bettercap/bettercap "bettercap") or use your own Access Point to redirect network traffic from the mobile device to your host machine (see below);
```

Comment:
Could you please correct "Access Point" to lowercase for all occurrences in this section? Thanks!
```suggestion
- Sometimes you need to execute a MITM attack to force the mobile device to talk to you. For this scenario you should consider [bettercap](https://github.com/bettercap/bettercap "bettercap") or use your own access point to redirect network traffic from the mobile device to your host machine (see below);
```

Reply:
New commit. Comment implemented.